src/minimega/vmtags.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (2015) Sandia Corporation.
+// Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
+// the U.S. Government retains certain rights in this software.
+
+// This is based on https://golang.org/src/text/template/parse/lex.go
+
+package main
```

Comment:
can we replace this whole file with just packing []*vmInfo into the Data field of the response?